src/pages/components/contained-list/usage.mdx

```diff
@@ -193,7 +163,33 @@ other text and components on the page.
 <Row>
 <Column colLg={8}>
 
-![Contained list on page variant in context image](images/contained-list-usage-7.png)
+![Contained list on page variant in context image](images/contained-list-usage-5.png)
+
+</Column>
+</Row>
+
+### Sizing
+
+The on page list comes in four height sizes — extra large (64px), large (48px),
+medium (40px), and small (32px). The large height size is the default size.
+
+<Row>
+<Column colLg={8}>
+
+![Height sizes for on page variant](images/contained-list-usage-6.png)
+
+</Column>
+</Row>
+
+The list header height is always fixed, while list item row heights can vary
+based on the amount of content in each row. The content in each row is flexible,
+but by default each list item row height follows the extra large, large or small
+height size for one line of content.
+
+<Row>
+<Column colLg={8}>
+
+![Row content size for on page variant](images/contained-list-usage-7.png)
 
```

Comment:
Small detail, but it would be nice to add the fixed height indication for the list header.

Reply:
I will add that for both the on-page and disclosed variant images. 👍